test/sqllogictest/transform/dataflow.slt

```diff
@@ -62,8 +62,12 @@ EXPLAIN PLAN FOR VIEW foo3
 | ArrangeBy (#0)
 
 %1 =
-| Join %0
+| Constant (6)
+
+%2 =
+| Join %0 %1 (= #0 #1)
 | | implementation = IndexedFilter #0 = 6
```

Comment:
I think the current old EXPLAIN output path might be missing the constant input part that you add here.